layouts/partials/contact-identifier.html

```diff
@@ -39,7 +39,7 @@
     <nav class="grid-row grid-gap identifier" aria-label="{{ $translation.footer.identifier_link_aria }}">
       <div class="tablet:grid-col-3">
         <ul class="usa-list usa-list--unstyled identifier-links">
-          <li><a href="https://www.gsa.gov/about-us">{{ $translation.footer.identifier_link_1 }}</a></li>
+          <li><a href="{{ .Permalink }}/about">{{ $translation.footer.identifier_link_1 }}</a></li>
```

Comment:
I think we need to replace permalink with baseurl. The link is not working on the translations as expected and it's getting a 404.